src/api/unified/symbol_manager.cpp

```diff
@@ -133,10 +135,11 @@ LibHandle openDynLibrary(const af_backend bknd_idx) {
     typedef af_err (*func)(int*);
 
     LibHandle retVal = nullptr;
-    for (size_t i = 0; i < extent<decltype(paths)>::value; i++) {
-        AF_TRACE("Attempting: {}", paths[i]);
-        if ((retVal = loadLibrary(join_path(paths[i], bkndLibName).c_str()))) {
-            AF_TRACE("Found: {}", join_path(paths[i], bkndLibName));
+    for (size_t i = 0; i < extent<decltype(pathPrefixes)>::value; i++) {
+        AF_TRACE("Attempting: {}", pathPrefixes[i]);
```

Comment:
Nice to have:

If the string is empty then change the message to:

Attempting: Default System Paths

Reply:
Sample output after change
```
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from Info
[ RUN      ] Info.All
[unified][002467] [ ../src/api/unified/symbol_manager.cpp:139 ] Attempting: Default System Paths
[unified][002467] [ ../src/api/unified/symbol_manager.cpp:159 ] Failed to load libafcpu.so.3: cannot open shared object file: No such file or directory
[unified][002467] [ ../src/api/unified/symbol_manager.cpp:139 ] Attempting: .
[unified][002467] [ ../src/api/unified/symbol_manager.cpp:159 ] Failed to load ./libafcpu.so.3: cannot open shared object file: No such file or directory
[unified][002467] [ ../src/api/unified/symbol_manager.cpp:139 ] Attempting: ./src/backend/cpu
[unified][002467] [ ../src/api/unified/symbol_manager.cpp:159 ] Failed to load ./src/backend/cpu/libafcpu.so.3: cannot open shared object file: No such file or directory
```